lib/middleware.js

```diff
@@ -94,8 +102,9 @@ class Kevin {
      * @param {array} configs — array of webpack config objects
      * @returns {string|null} - the name of the config, or null if there is none
      */
-    getConfigForAssetName(assetName, configs) {
-        const configName = this.entryMap[assetName];
+    getConfigForAssetName(reqPath, assetName, configs) {
```

Comment:
tempted to just make this the whole request object - unsure so starting a thread for discussion

Reply:
for now, I'd lean towards explicitly passing in only what we need

Reply:
k leaving for now